rust/connlib/clients/shared/src/lib.rs

```diff
@@ -60,28 +60,28 @@ impl Session {
         Self { channel: tx }
     }
 
-    /// Attempts to reconnect a [`Session`].
+    /// Reset a [`Session`].
     ///
-    /// Reconnecting a session will:
+    /// Resetting a session will:
     ///
     /// - Close and re-open a connection to the portal.
```

Comment:
If the network listening code fires spurious signals and the Client calls `reset` 2 or 3 times in a row, will that hurt anything? Does it need to be debounced?

Reply:
It doesn't really hurt, we just cycle through a couple of sockets.